html/befuncs/crypt.php

```diff
@@ -0,0 +1,22 @@
+<?php
+	// generate with openssl genpkey -algorithm rsa -outform PEM -out [path-to-key].pem
+	const KEY_FILE = '/vagrant/key.pem';
```

Comment:
/vagrant/ doesn't exist in a life environment. Use relative paths instead

Reply:
Where would you put the key instead.

Reply:
idk something like `../../key.pem`